composer.json

```diff
@@ -10,7 +10,7 @@
 	"require-dev": {
 		"squizlabs/php_codesniffer": "~2.2",
 		"phpmd/phpmd": "~2.3",
-		"ockcyp/covers-validator": "~0.4.0"
+		"phpunit/phpunit": "~5.4"
```

Comment:
It should be compatible with PHP 5.6, I think. PHP 5.5 requirement is obsolete.
See: https://github.com/wmde/FundraisingStore/pull/87